doc/pairs.md

```diff
@@ -0,0 +1,35 @@
+# Pairs examples
+
+## Lua Fun
+
+Queries using pairs support the [Lua Fun](https://github.com/luafun/luafun) library. Some examples of working with basic functional functions below.
```

Comment:
I think that we have to show more examples that shows `crud.pairs`, not `luafun`.

First of all, we need to show difference between calls with and without `use_tomap`. Moreover, I would like to see returned data, just like in `select` examples.

We also need to show pairs with pagination (be careful, reverse pagination isn't supported here).

BTW, what do you think about showing select without conditions before all examples to understand space contents. It can be helpful to understand filters and pagination.